webui/react/src/pages/ExperimentList.tsx

```diff
@@ -339,12 +358,12 @@ const ExperimentList: React.FC = () => {
         </TableBatch>
         <Table
           columns={columns}
-          dataSource={filteredExperiments}
+          dataSource={experiments}
           loading={{
             indicator: <Indicator />,
-            spinning: !experimentsResponse.hasLoaded,
+            spinning: false,
```

Comment:
do we still want to track this?

Reply:
ah yes, I temporarily did it but forgot to add a TODO comment to remind me. thanks!